mifospay/src/main/AndroidManifest.xml

```diff
@@ -35,7 +35,8 @@
         <activity
             android:name=".auth.ui.LoginActivity"
             android:theme="@style/AppTheme"
-            android:windowSoftInputMode="stateVisible" />
+            android:windowSoftInputMode="stateVisible"
+            android:noHistory="true" />
```

Comment:
@Harshad09 I don't think it's a good idea to use `noHistory` as we are also creating intent to `MobileVerificationActivity` from this activity from which we may want to return.

Instead, you can probably have a workaround to handle the back press in passcode activity